src/libraries/System.Threading.AccessControl/src/System/Security/AccessControl/MutexSecurity.cs

```diff
@@ -121,9 +121,9 @@ internal MutexSecurity(SafeWaitHandle handle, AccessControlSections includeSecti
             // Let the underlying ACL API's demand unmanaged code permission.
         }
 
-        private static Exception HandleErrorCode(int errorCode, string name, SafeHandle handle, object context)
+        private static Exception? HandleErrorCode(int errorCode, string? name, SafeHandle? handle, object? context)
```

Comment:
Which one allows a null name?  Passing `null` with `new MutexSecurity(null, AccessControlSections.Whatever)` will throw an exception.

Reply:
OK, then never mind, i was talking about the same contractor you mentioned, just because base constructor was allowing null `name` and there is no other operation. `base(true, ResourceType.KernelObject, name, includeSections, HandleErrorCode, null)`